src/common/crc32c_s390x.c

```diff
@@ -1,14 +1,29 @@
 /*
- * CRC-32 algorithm implemented with the z/Architecture
- * Vector Extension Facility.
+ * CRC-32 algorithm implemented with the z/Architecture Vector Extension
+ * Facility.
+ *
+ * Copyright 2024 IBM Corporation
```

Comment:
This is rewording existing `Copyright IBM Corp. 2024` (and same for 2015 in the other files).  Is amending copyright notices like that OK from a legal perspective?

Reply:
Yes, I think so. It's to be consistent with the rest of the project.

Reply:
For files under `src/arch`, `Copyright (C) 2024 IBM, Inc.` is added, so we aren't consistent even within this PR ;)

Reply:
Blah.

Reply:
FWIW there is still a (very minor) mismatch involving `(C)`:
```
+ * Copyright (C) 2024 IBM Corporation
+ * Copyright (C) 2024 IBM Corporation
+ * Copyright 2024 IBM Corporation
+ * Copyright 2024 IBM Corporation
+ * Copyright 2015 IBM Corporation
+ * Copyright 2015 IBM Corporation
```

Reply:
fixed